### PR TITLE
Refactor parameters for the usage of GMT extension 

### PR DESCRIPTION
**Pull request by ChengnanShi-Work**
Hi Dr.Hu @JayHuLBL ,for the extension usage of GMT, I have made two minor changes for an existing example model Experimental.DHC.CentralPlants.Example.Plant.mo in MBL:

   1. Add "replaceable" prefix for two parameters ("perChi", "pla") for new type redeclaring.
   2. Add two more parameters ("dTApp" and "PFan_nominal") because they are also important parameters of the cooling plant which we wish to replace value for.

### CODE REVIEW

File: Buildings/Experimental/DHC/CentralPlants/Cooling/Examples/Plant.mo [hunk 1]
```diff
@@ -160,12 +164,16 @@ Buildings.Experimental.DHC.CentralPlants.Cooling.Plant</a>.
       revisions="<html>
 <ul>
 <li>
+October 20, 2021, by Chengnan Shi:<br/>
+Revise the model for extensibility.
```
Comment (JayHuLBL) [inline Buildings/Experimental/DHC/CentralPlants/Cooling/Examples/Plant.mo:168]: change the revision note as, `Revised the model for extensibility. This is for <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2749\">issue #2749</a>.`

**Review by JayHuLBL — CHANGES_REQUESTED** [review]
@ChengnanShi-Work Please also add the release note in `Buildings.UsersGuide.ReleaseNotes.Version_9_0_0`, section `<!-- Backward compatible changes -->`

File: Buildings/Experimental/DHC/CentralPlants/Cooling/Examples/Plant.mo (hunk 1 above)
Comment (ChengnanShi-Work) [reply Buildings/Experimental/DHC/CentralPlants/Cooling/Examples/Plant.mo:168]: Thank you Dr. Hu. Will do!

**Comment by ChengnanShi-Work** [comment]
> @ChengnanShi-Work Please also add the release note in `Buildings.UsersGuide.ReleaseNotes.Version_9_0_0`, section `<!-- Backward compatible changes -->`

Should I add the release note at the end (after Buildings.Utilities.Math) or follow the alphabetical order?

**Comment by mwetter** [comment]
> > @ChengnanShi-Work Please also add the release note in `Buildings.UsersGuide.ReleaseNotes.Version_9_0_0`, section `<!-- Backward compatible changes -->`
> 
> Should I add the release note at the end (after Buildings.Utilities.Math) or follow the alphabetical order?

Please add them in alphabetical order.

**Pull request merged**